scimono-compliance-tests/Readme.md

```diff
@@ -0,0 +1,78 @@
+# SCIM 2.0 Compliance tests
+
+A set of integration tests that are executed against external SCIM 2.0 provider implementation and verify that if it is compliant with the SCIM 2.0 standard.
+
+## Prerequisite
+
+### Required tools
+- JDK Version 1.8
+- Maven - version 3.x
+
+### Installation
+Build project **scimono-compliance-tests** with maven:
+
+```
+mvn clean install
+```
+
+## Run integration tests
+
+Just execute the shaded executable jar you previously built located in `scimono-compliance-tests/target`: 
+
+```
+java -jar scimono-compliance-tests-${version}-shaded.jar scim.service.url={Url of the scim endpoint of service provider}
```

Comment:
List an examle here. I think that the user will immediately wonder what URL exactly they should enter. Is it /scim or /scim/ or /scim/Users?

Reply:
done